manubot/filters.py

```diff
@@ -0,0 +1,15 @@
+from random import Random, shuffle
+
+
+def mb_random(value, seed=None):
+    shuffled = list(value)
+    if seed is not None:
+        Random(seed).shuffle(shuffled)
+    else:
+        shuffle(shuffled)
+    return shuffled
```

Comment:
```suggestion
    shuffled = list(value)
    kwargs = {} if seed is None {"seed": seed}:
    Random(**kwargs).shuffle(shuffled)
    return shuffled
```

Reply:
oh - wow - this is really elegant and allows other args too!

Reply:
Actually Random takes only seed and not kwargs. I have a shorter implementation inbound.

Reply:
Ah `*args`

Reply:
I'm not sure it makes sense to do args (although that's what I did the first time) because as far as I can tell the constructor takes no other types of args: https://docs.python.org/3/library/random.html#random.Random